File: apps/backend-node/src/features/hosts/dto/host.dto.ts
import type { HostStatus } from './host-status.dto';

export interface HostDto {
  id: string;
  profileId: string;
  displayName: string;
  status: HostStatus;
  verifiedAt: string | null;
  createdAt: string;
  updatedAt: string;
}

File: apps/backend-java/src/main/java/com/housingplatform/features/hosts/dto/HostDto.java
package com.housingplatform.features.hosts.dto;

import com.housingplatform.persistence.enums.HostStatus;
import java.time.OffsetDateTime;
import java.util.UUID;

public record HostDto(
    UUID id,
    UUID profileId,
    String displayName,
    HostStatus status,
    OffsetDateTime verifiedAt,
    OffsetDateTime createdAt,
    OffsetDateTime updatedAt) {}
